meta-intel-edison-bsp/recipes-kernel/linux/linux-yocto_5.15.81.bb

```diff
@@ -82,9 +81,13 @@ SRC_URI:append = " file://0001-tty-tty_io-Switch-to-vmalloc-fallback-in-case-of-
 # usefull kernel debug options here
 #SRC_URI:append = " file://0001-8250_mid-toggle-IO7-on-ttyS1-interrupt-entry.patch"
 
-SRCREV ??= "v${PV}"
+SRCREV = "e4a7232c917cd1b56d5b4fa9d7a23e3eabfecba0"
```

Comment:
But this is a bit unflexible, can we use some variable or tag or so here?

Reply:
Like you say further on: github reviewing is a bit meh. So this file 5.15.81 is dropped further on. That said, in _6.1.y.bb we have:
```
SRCREV = "v${LINUX_VERSION}"
LINUX_VERSION_EXTENSION = "-edison-acpi-${LINUX_KERNEL_TYPE}"
PV = "${LINUX_VERSION}+git${SRCPV}"
LINUX_VERSION ?= "6.1.55"
```
With past Yocto versions  the "_.5.15.81" in the filename would automatically be in $PV, then PV would be used to create SRVREV. That leads to some circular reference currently. The above solution seems to work, and is indeed easier to update in the future. I will fix this in _6.1.y-rt.bb, _6.6.0.bb.